avocado/core/job.py

```diff
@@ -167,10 +167,7 @@ def __start_job_logging(self):
 
         # TODO: Fix this, this is one of the few cases where using the config
         # generated from the new settings with a hardcoded 'default' value
-        try:
-            store_logging_stream = self.config.get('run.store_logging_stream', [])
-        except AttributeError:
-            store_logging_stream = []
+        store_logging_stream = self.config.get('run.store_logging_stream', [])
```

Comment:
And do we need the list, I mean, shouldn't this be:

```python
store_logging_stream = self.config.get('run.store_logging_stream')
```

Reply:
I got some problems in the past where this was not registered and was returning None in this case.

Reply:
Yes, that was my point: I was expecting it to be registered and had a default such as `[]`.  But that's not a big deal, let's move this forward.

Reply:
> Yes, that was my point: I was expecting it to be registered and had a default such as `[]`. But that's not a big deal, let's move this forward.

On, register with a default value of `[]` really makes sense!